SETUP_GUIDELINES.md

```diff
@@ -0,0 +1,34 @@
+## Steps for Developers 
+1. Download the latest stable release of [Android Studio](https://developer.android.com/studio/index.html).
+2. Fork the [Project Repository](https://github.com/openfoodfacts/openfoodfacts-androidapp) by clicking "Fork".
+3. Clone the repository `git clone https://github.com/YOUR_USERNAME/openfoodfacts-androidapp.git`.
+4. Initialize Git `git init`.
+5. Run the project in Android Studio 
+6. Fetch the latest version of the code from branch "master".
```

Comment:
If the user cloned the repo it automaticaly contains the latest commits. you dont need to fetch again and again. you can remove this one.

Reply:
Thanks !